src/site/content/en/blog/launch-handler/index.md

```diff
@@ -0,0 +1,244 @@
+
+### Enabling via about://flags
+
+To experiment with the Launch Handler API locally, without an origin trial token, enable the
+`#enable-desktop-pwas-launch-handler` flag in `about://flags`.
+
+### Enabling support during the origin trial phase
+
+Starting in Chromium&nbsp;98, the Launch Handler API will be available as an origin trial in
+Chromium. The origin trial is expected to end in Chromium&nbsp;102 (June&nbsp;15, 2022).
+
+{% include 'content/origin-trials.njk' %}
+
+### Register for the origin trial {: #register-for-ot }
+
+{% include 'content/origin-trial-register.njk' %}
+
+### Feature detection
+
+To check if the Launch Handler API is supported, use:
+
+```javascript
+if ('launchQueue' in window && 'targetURL' in LaunchParams.prototype) {
+  // The Launch Handler API is supported.
+}
+```
+
+### The `launch_handler` manifest member
+
+To declaratively specify the launch behavior of your app, add the `launch_handler` manifest member
+to your manifest. It has two sub-fields, `route_to` and `navigate_existing_client`. The former lets
+you control whether a new or an existing client should be launched, and the latter how and if this
+client should be navigated. The code snippet below shows an excerpt of a Web App Manifest file with
+exemplary values that would always route all launches to a new client.
+
+```json
+{
+  "launch_handler": {
+    "route_to": "new-client",
+    "navigate_existing_client": "always"
+  }
+}
+```
+
+If unspecified, `launch_handler` defaults to
+`{"route_to": "auto", "navigate_existing_client": "always"}`. The allowed values for the sub-fields
+are as follows:
+
+- `route_to`:
+
+  - `new-client`: A new browsing context is created in a web app window to load the launch's target
+    URL.
+  - `existing-client`: The most recently interacted with browsing context in a web app window for
+    the app being launched is chosen to handle the launch. How the launch is handled within that
+    browsing context depends on `navigate_existing_client`.
+  - `auto`: The behavior is up to the user agent to decide what works best for the platform. For
+    example, mobile devices only support single clients and would use `existing-client`, while
+    desktop devices support multiple windows and would use `new-client` to avoid data loss.
+
+- `navigate_existing_client`:
+  - `always`: Existing browsing contexts chosen for launch will navigate the browsing context to the
+    launch's target URL.
+  - `never`: Existing browsing contexts chosen for launch will not be navigated and instead have
+    `targetURL` in the enqueued `LaunchParams` set to the launch's target URL.
+
+Both `route_to` and `navigate_existing_client` also accept a list (array) of values, where the first
+valid value will be used. This is to allow new values to be added to the spec without breaking
+backwards compatibility with existing implementations.
+
+For example, if the value `"matching-url-client"` were added, sites would specify
```

Comment:
```suggestion
For example, if the hypothetical value `"matching-url-client"` were added, sites would specify
```

Reply:
I don't like adding adjectives, but I wanted to make it clearer that this is not a promise to add this value.